packages/valory/connections/test_tools/docker_images.py

```diff
@@ -0,0 +1,191 @@
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""ACN Docker Image."""
+
+import logging
+import socket
+import time
+from typing import Dict, List
+
+from docker import DockerClient  # pylint: disable=import-error
+from docker.models.containers import Container  # pylint: disable=import-error
+
+from aea.exceptions import enforce
+from aea.test_tools.docker_image import DockerImage
+
+
+LOCAL_ADDRESS = "0.0.0.0"
+
+# created agent: bootstrap_peer
+#     private key: 7f669ab5eee5719e385f7aeb1973769fc75b7cbbe0850ca16c4eabe84e01afbd
+#     public key:  0270475f9b78c0285a6ac6067582f5e159ec147ccb03aee16a32731f68920b1ae8
+#     PeerID:      16Uiu2HAm2yxmLQTZTrxjo5c4k5ka8AVMcpeD5zMMeasE6xDw1YQw
+#
+# created agent: entry_node_1
+#     private key: 5b8d3be9f27489040a01adc2b746b9a5f9d32ed843d99cf7d2995c8140636190
+#     public key:  02197b55d736bd242311aaabb485f9db40881349873bb13e8b60c8a130ecb341d8
+#     PeerID:      16Uiu2HAkw99FW2GKb2qs24eLgfXSSUjke1teDaV9km63Fv3UGdnF
+#
+# created agent: entry_node_2
+#     private key: cc096b7be575c11d3d3d2f8a9c9be9bd59b351317b2a114ef7e014cc5a92508e
+#     public key:  0287ee61e8f939aeaa69bd7156463d698f8e74a3e1d5dd20cce997970f13ad4f12
+#     PeerID:      16Uiu2HAm4aHr1iKR323tca8Zu8hKStEEVwGkE2gtCJw49S3gbuVj
+
+
+GENESIS_MADDR = (
+    "/dns4/0.0.0.0/tcp/10000/p2p/16Uiu2HAm2yxmLQTZTrxjo5c4k5ka8AVMcpeD5zMMeasE6xDw1YQw"
+)
+
+BOOTSTRAP: Dict[str, str] = dict(
+    AEA_P2P_ID="7f669ab5eee5719e385f7aeb1973769fc75b7cbbe0850ca16c4eabe84e01afbd",
+    AEA_P2P_URI_PUBLIC=f"{LOCAL_ADDRESS}:9000",
+    AEA_P2P_URI=f"{LOCAL_ADDRESS}:10000",
+    AEA_P2P_DELEGATE_URI=f"{LOCAL_ADDRESS}:11000",
+    AEA_P2P_URI_MONITORING=f"{LOCAL_ADDRESS}:8080",
+    ACN_LOG_FILE="/acn/libp2p_node.log",
+)
+
+NODE1: Dict[str, str] = dict(
+    AEA_P2P_ID="5b8d3be9f27489040a01adc2b746b9a5f9d32ed843d99cf7d2995c8140636190",
+    AEA_P2P_URI_PUBLIC=f"{LOCAL_ADDRESS}:9001",
+    AEA_P2P_URI=f"{LOCAL_ADDRESS}:10001",
+    AEA_P2P_DELEGATE_URI=f"{LOCAL_ADDRESS}:11001",
+    AEA_P2P_URI_MONITORING=f"{LOCAL_ADDRESS}:8081",
+    AEA_P2P_ENTRY_URIS=GENESIS_MADDR,
+    ACN_LOG_FILE="/acn/libp2p_node.log",
+)
+
+
+NODE2: Dict[str, str] = dict(
+    AEA_P2P_ID="cc096b7be575c11d3d3d2f8a9c9be9bd59b351317b2a114ef7e014cc5a92508e",
+    AEA_P2P_URI_PUBLIC=f"{LOCAL_ADDRESS}:9002",
+    AEA_P2P_URI=f"{LOCAL_ADDRESS}:10002",
+    AEA_P2P_DELEGATE_URI=f"{LOCAL_ADDRESS}:11002",
+    AEA_P2P_URI_MONITORING=f"{LOCAL_ADDRESS}:8082",
+    AEA_P2P_ENTRY_URIS=GENESIS_MADDR,
+    ACN_LOG_FILE="/acn/libp2p_node.log",
+)
```

Comment:
hardcoded here for convenience as I'm attempting to make this work. Tests can run with the bootstrap (genesis) node only, however the entry peer nodes don't run yet.